src/Login/Login.js

```diff
@@ -3,35 +3,29 @@ import PropTypes from 'prop-types';
 import './Login.css';
 
 class Login extends Component {
-  runChange = (event) => {
+
+  runChange = async (event) => {
     event.preventDefault()
-    this.submitLogin()
-    this.props.action()
+    const user = await this.submitLogin()
+    this.props.action(user)
```

Comment:
Action is a pretty vague name, rename the prop to indicate what the action is performing!